runtime/doc/options.txt

```diff
@@ -5415,39 +5414,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	See |shell-unquoting| which talks about separating this option into 
 	multiple arguments.
 	This option cannot be set from a |modeline| or in the |sandbox|, for
-	security reasons.
-
-						*'shellpipe'* *'sp'*
-'shellpipe' 'sp'	string	(default ">", ">%s 2>&1", "| tee", "|& tee" or
```

Comment:
Related: https://github.com/neovim/neovim/pull/16271

Reply:
Yes, as it turns out powershell was the reason for one of the failing tests (really great that that test exists, even if it was a little too brittle due to parsing output).